conf/container_repo.yaml.template

```
@@ -0,0 +1,16 @@
+# Container registry and repo set information to use for container repo syncing.
+CONTAINER_REPO:
+  registries:
+    redhat:
+      URL: https://registry.redhat.io
+      USERNAME:
+      PASSWORD:
+      REPOS_TO_SYNC: [rhel7, rhscl/mysql-57-rhel7, rhscl/ruby-24-rhel7]
```

Spacing may be off, since I'm on my phone, but I'd prefer this list form
```suggestion
      REPOS_TO_SYNC:
        - rhel7
        - rhscl/mysql-57-rhel7
        - rhscl/ruby-24-rhel7]
```